packages/hint/docs/user-guide/concepts/connectors.md

```diff
@@ -17,27 +17,15 @@ make it look like the following:
 
 Where `connectorName` is the name of the connector.
 
-## Built-in connectors and platform support
+## Official connectors and platform support
 
 All the built-in `connector`s run in any of the supported platforms:
 Linux, macOS, and Windows. The only caveat is that, for the `connector`
 that you specify in the`.hintrc` file, you will need to have the
 browser the `connector` is for installed as `webhint` will not
 install it for you.
```

Comment:
Seems this won't apply to any connectors other than `connector-puppeteer` in the near future. Think we should remove or reword this section?

Reply:
Maybe when we deprecate `connector-chrome`?

Reply:
Seems reasonable.